nano/node/active_transactions.cpp

```diff
@@ -382,7 +382,7 @@ void nano::active_transactions::cleanup_election (nano::unique_lock<nano::mutex>
 			node.network.publish_filter.clear (block);
 		}
 	}
-	node.logger.try_log (boost::str (boost::format ("Election erased for root %1%") % election.qualified_root.to_string ()));
+	node.logger.try_log (boost::str (boost::format ("Election erased for root %1%, confirmed: %2$b") % election.qualified_root.to_string () % election.confirmed ()));
```

Comment:
There is a strange $b that looks wrong.

Reply:
this turns the 1/0 into true/false (from boost::format)

Reply:
Yeap, section "printf format specifications" in
https://www.boost.org/doc/libs/1_66_0/libs/format/doc/format.html